submissions/kazeemErinfolami/html/index.html

```diff
@@ -11,8 +11,8 @@ <h1><a href="#">Emarket</a></h1>
       </div>
       <nav>
         <ul>
-          <li><a href="#">Cart</a></li>
-          <li><a href="#">Sign In</a></li>
+          <li><a href="../htmlForms/contact.html">Contact Form</a></li>
+          <li><a href="../htmlForms/signUp.html">SignUp</a></li>
```

Comment:
Do you have a signup page?

Reply:
Yes, but thought you asked me to remove it

Reply:
Hope everything is fine here, can I move on to the next project?